components/uArchARM/MapTable.cpp

```diff
@@ -0,0 +1,28 @@
+#include <algorithm>
```

Comment:
Need the copyright boilerplate on top of the file.

Reply:
Fixed